### PR TITLE
x11-misc/adriconf Advanced DRI Configurator

### PR DESCRIPTION
**Pull request by mercuriete**
…gure open source graphics drivers

Closes: https://bugs.gentoo.org/697676
Signed-off-by: Abel Garcia Dorta <mercuriete@yahoo.es>

### CODE REVIEW

**Comment by mercuriete** [comment]
This PR is almost a copy from
https://github.com/gentoo-mirror/poly-c/tree/master/x11-misc/adriconf

and ported to the new repository and fixing some errors.

**Comment by thesamesam** [comment]
@mercuriete Please ask @Polynomial-C if he is okay with you adding it (and ideally if he could commit it).

**Comment by mercuriete** [comment]
@thesamesam I need to bump it first to 2.4.1.
https://gitlab.freedesktop.org/mesa/adriconf/-/releases/v2.4.1

and then ask to @Polynomial-C

File: x11-misc/adriconf/metadata.xml
Comment (Polynomial-C) [inline x11-misc/adriconf/metadata.xml:7]: `desktop-misc` project has been disbanded. See https://marc.info/?l=gentoo-dev&m=160642206127820&w=4

Comment (mercuriete) [reply x11-misc/adriconf/metadata.xml:7]: hi @Polynomial-C can you take care of a similar PR?
I don't know how to continue.

I don't know how to give you credit in this PR. All this code was copied from your overlay.

thanks :)

File: x11-misc/adriconf/adriconf-2.4.ebuild
Comment (ConiKost) [inline x11-misc/adriconf/adriconf-2.4.ebuild:4]: Sorry for rotting this PR. Could you update to EAPI=8?

File: x11-misc/adriconf/adriconf-2.4.ebuild
Comment (ConiKost) [inline x11-misc/adriconf/adriconf-2.4.ebuild:9]: This seems dead, as it's marked read-only. But you seems to use anyway the newer fork by freedesktop.org?

File: x11-misc/adriconf/adriconf-2.4.ebuild
Comment (ConiKost) [inline x11-misc/adriconf/adriconf-2.4.ebuild:14]: Could you update to 2.4.1?

File: x11-misc/adriconf/adriconf-2.4.ebuild
Comment (ConiKost) [inline x11-misc/adriconf/adriconf-2.4.ebuild:20]: No new line please.

File: x11-misc/adriconf/adriconf-2.4.ebuild
Comment (ConiKost) [inline x11-misc/adriconf/adriconf-2.4.ebuild:64]: This is depreated. Please use ```/usr/share/metainfo```.

File: x11-misc/adriconf/adriconf-2.4.ebuild
Comment (ConiKost) [inline x11-misc/adriconf/adriconf-2.4.ebuild:72]: Please add the missing inherit ```xdg-utils```.

File: x11-misc/adriconf/adriconf-2.4.ebuild
Comment (ConiKost) [inline x11-misc/adriconf/adriconf-2.4.ebuild:35]: This needs to be updated to ```[egl(+)]```, as newest mesa release in tree dropped this use flag and enables this always.

File: x11-misc/adriconf/metadata.xml
Comment (ConiKost) [reply x11-misc/adriconf/metadata.xml:7]: I would be willing to steppin as maintainer.

**Review by ConiKost — CHANGES_REQUESTED** [review]

**Comment by mercuriete** [comment]
Hey @ConiKost 
You seems to have more knowledge than me.
Can you take over this PR or create a new one?
I will try to address all problems you pointed out but I don't know if I am capable to do It.

Thanks for the feedback, I will try to give It some time on this weekend.

**Comment by mercuriete** [comment]
@ConiKost I rebased this PR and fixed the egl(+) problem...
can you review against the new file?
I had to create a new file called 2.4.1 and all your comments now are outdated.

thanks.

**Comment by mercuriete** [comment]
I just copied again the files from @Polynomial-C 
https://github.com/gentoo-mirror/poly-c/tree/master/x11-misc/adriconf

Please somebody with more knowledge than me try to push this forward.

File: x11-misc/adriconf/adriconf-2.4.1.ebuild
Comment (juippis) [inline x11-misc/adriconf/adriconf-2.4.1.ebuild:29]:
```
CMake Error at /usr/lib64/cmake/Boost-1.77.0/BoostConfig.cmake:141 (find_package):
  Could not find a package configuration file provided by "boost_locale"
  (requested version 1.77.0) with any of the following names:

    boost_localeConfig.cmake
    boost_locale-config.cmake

  Add the installation prefix of "boost_locale" to CMAKE_PREFIX_PATH or set
  "boost_locale_DIR" to a directory containing one of the above files.  If
  "boost_locale" provides a separate development package or SDK, be sure it
  has been installed.
Call Stack (most recent call first):
  /usr/lib64/cmake/Boost-1.77.0/BoostConfig.cmake:258 (boost_find_component)
  /usr/share/cmake/Modules/FindBoost.cmake:594 (find_package)
  CMakeLists.txt:47 (find_package)


-- Configuring incomplete, errors occurred!
```
This requires `boost[nls]`.

File: x11-misc/adriconf/adriconf-2.4.1.ebuild
Comment (juippis) [inline x11-misc/adriconf/adriconf-2.4.1.ebuild:19]:
```suggestion
LICENSE="GPL-3+"
```
> This program is free software: you can redistribute it and/or modify it under the terms of the GNU General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.

File: x11-misc/adriconf/adriconf-2.4.1.ebuild
Comment (juippis) [inline x11-misc/adriconf/adriconf-2.4.1.ebuild:22]: This USE flag does nothing - did you intend to put some configure option for it?

**Review by juippis — COMMENTED** [review]
(Comments apply to -9999 ebuild as well)

Would you be willing to become a proxied maintainer for this, or will you leave it to @conikost 100 %? Since we can't add new packages without maintainers as a general rule.

**Comment by mercuriete** [comment]
Hi all I don't have time and knowledge to move this forward.

Thank you for your time to every people involved in this thread.

@ConiKost Could you open a new PR instead of this one? Thank you very much.

**Pull request closed without merging**
